pumpkin-lib/src/api/outputs/mod.rs

```diff
@@ -24,6 +24,7 @@ pub enum SatisfactionResult {
 }
 
 /// The result of a call to [`Solver::satisfy_under_assumptions`].
+#[allow(clippy::large_enum_variant)]
```

Comment:
I don't understand this, on any other branch this is not a problem. Note that GitHub actions always runs the latest stable version, and clippy and formatting are run on the latest nightly.

Reply:
I am not quite sure either, could perhaps be a clippy bug? It was also necessary for the `SatisfactionResultUnderAssumptions` previously